### DISCUSSION TITLE
Sensor deployment limit?

### DISCUSSION THREAD

**Discussion by Tredmond7511** [Security-Onion-Solutions/securityonion#10647] [Unsupported Versions]
My organization is thinking of deploying 40 forward sensors to 1 manager node with separate search node. It will be heavy traffic on each sensor. Is there a limit to which the manager or search node can handle? Also what estimated tech specs should be configured on the manager and search nodes for optimal performance in this setup?

**Comment by dougburks** [comment]
There are deployments out there with a higher number of sensors. The main thing is making sure that you adequately size the sensors, manager, and search node(s):
https://docs.securityonion.net/en/2.3/hardware.html

In case you weren't aware, we sell hardware appliances to make this much simpler:
https://securityonionsolutions.com/hardware/

And if you're not able to purchase our hardware, we also sell professional services to help you get the most out of your own hardware:
https://securityonionsolutions.com/support

**Comment by Jaap79** [comment]
Hi @Tredmond7511 

We're running 80+ sensors right now, so far without any issues. What might help in your situation, if you're worried about bandwidth, is to add a Receiver Node (or 2?) into the environment. That should lighten the load on the Manager itself.

**Reply by cedgman** [reply]
Wow 80+ sensors!  That's amazing.. how many search nodes and receiver nodes do you have for such an environment?

**Reply by Jaap79** [reply]
5 search nodes with a few TB storage each, and 2 receiver nodes 🥳

**Reply by atndon** [reply]
Thank you for sharing your experience. Can you share your insight about data retention too? For how long are you keeping the data so we can understand if we need to throw additional TBs into the design?

Thanks again
